UefiPayloadPkg/NetworkDrivers/build-ipxe.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+rm -rf ipxe/
+git clone git@github.com:Dasharo/ipxe.git -b v1.21.1
```

Comment:
This is very old tag... Nearly 1,5 years old. Can't we use something relatively newer? coreboot just checks out the most recent commit ID when bumping stable version. Also this tag may cause problems depending on the coreboot-sdk used. The `make` in this script will also use a different compiler than the one used by coreboot.

Reply:
@miczyg1  So, how to solve that? AFAIK iPXE is right now added as binary, shall we build iPXE inside coreboot-sdk container?

Reply:
Yes, and I did implement it already:
https://github.com/Dasharo/edk2/pull/34
https://github.com/Dasharo/coreboot/pull/259/files